Carmunication/Controllers/SessionList/GroupAdd/SelectDetailPointMapViewController.swift

```diff
@@ -112,12 +120,14 @@ extension SelectDetailPointMapViewController: NMFMapViewCameraDelegate {
                 y: mapView.frame.size.height / 2
             )
         )
-
+        print("mapViewCameraIdle 내부")
```

Comment:
이렇게 작성하는 방법도 있습니다 #line도 있고 #file도 있어요

```suggestion
        print("\(#function) 내부")
```

Reply:
오호! 꿀팁입니다...

Reply:
이런 방법도 있었군요